README.md

```diff
@@ -227,6 +227,19 @@ The below example is for v1
 client.update_service service1
 ```
 
+#### Patch an entity (by name)
+For example: `patch_service "service name"`, `patch_pod "pod name"`, `patch_replication_controller "rc name"`, `patch_secret "secret name"`, `patch_resource_quota "resource quota name"`, `patch_limit_range "limit range name"`, `patch_persistent_volume "persistent volume name"`, `patch_persistent_volume_claim "persistent volume claim name"`, `patch_component_status "component name"`, `patch_service_account "service account name"`
```

Comment:
this section here is incorrect - the patch methods receive as a bare minimum 2 parameters, potentially 3. so the examples above are misleading. Also this section seems a bit unnecessary and too long.